### PR TITLE
MWPW-136767 - disable scroll

### PR DESCRIPTION
**Pull request by saragajic11**
This PR has been opened because previous PR (https://github.com/adobecom/milo/pull/1419) contains some unwanted changes on footer's appearance on test pages, while code doesn't have any changes which could cause this behaviour.

This PR should resolve the issue of scrollable background when modal dialog (which is authored to have a 'curtain') is opened. In other cases (with 'curtain-off') background should be scrollable per requirements.

Resolves: [MWPW-136767](https://jira.corp.adobe.com/browse/MWPW-136767)

Test URLs:

Before: https://main--milo--adobecom.hlx.page/drafts/rclayton/video-examples?martech=off
After: https://mwpw-136767-disable-scroll--milo--saragajic11.hlx.page/drafts/rclayton/video-examples?martech=off

Commerce dialog (it should work the same as before this PR):

Before: https://main--milo--adobecom.hlx.page/drafts/ruchika/discover1?martech=off
After: https://mwpw-136767-disable-scroll--milo--saragajic11.hlx.page/drafts/ruchika/discover1?martech=off

### CODE REVIEW

**Review by Blainegunn — APPROVED** [review]

**Comment by Blainegunn** [comment]
thank you @saragajic11

**Comment by Ruchika4** [comment]
@saragajic11 could you please add the required label to the PR.
@spadmasa , thanks for testing the PR https://github.com/adobecom/milo/pull/1419 Could you please add your comments regarding CC page verification here too. Footer issue you noticed in previous PR should be resolved here.

**Comment by Ruchika4** [comment]
@saragajic11 also please assign this PR to a QE or SME from DC side to verify.

**Comment by spadmasa** [comment]
Verified the flow in cc in the below url scrollable background when modal dialog is opened issue is seen fixed .
https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=mwpw-136767-disable-scroll--milo--saragajic11#after-effects
image

but on the content in the page still see the issue.
image
image - transperancy on the scroll
share icons missing in the footer 
image

issues are not seen in main branch : https://main--cc--adobecom.hlx.page/creativecloud/animation/discover

**Comment by Ruchika4** [comment]
> Verified the flow in cc in the below url scrollable background when modal dialog is opened issue is seen fixed . https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=mwpw-136767-disable-scroll--milo--saragajic11#after-effects image
> 
> but on the content in the page still see the issue. image image - transperancy on the scroll share icons missing in the footer image
> 
> issues are not seen in main branch : https://main--cc--adobecom.hlx.page/creativecloud/animation/discover
> 
> Thankyou @Ruchika4 , verified the flow in mobile mode as well as advised and do not see the scroll functionality working https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=mwpw-136767-disable-scroll--milo--saragajic11#animate image image

Thanks for testing @spadmasa 
@saragajic11 looks like there is some issues with sending viewport dimensions to iframe (commerce modal) which is causing it to break in all viewports. I tested it in a test branch by removing body css file and just defining the css class name .disable-scroll Test page here https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=disable-scroll--milo--ruchika4#after-effects and it appears to be working fine for commerce modal. @spadmasa could you also please verify this. I see this fixes some other content issue you pointed out earlier too.

**Comment by Ruchika4** [comment]
Commerce modal is used on homepage too so please get it tested by them also once. Our team (CC team) is testing only commerce modal on CC pages. So please get all the other modals (georouting etc) tested before getting the PR merged.

File: libs/blocks/modal/modal.css [hunk 1]
```diff
@@ -195,3 +191,7 @@
     height: 845px;
   }
 }
+
+body.disable-scroll {
```
Comment (Ruchika4) [inline libs/blocks/modal/modal.css:195]: Issues reported for commerce modal are getting fixed by using css class name without the tag name i.e

.disable-scroll @saragajic11 could you please take a look. Thanks

**Review by Ruchika4 — CHANGES_REQUESTED** [review]

**Comment by spadmasa** [comment]
> > Verified the flow in cc in the below url scrollable background when modal dialog is opened issue is seen fixed . https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=mwpw-136767-disable-scroll--milo--saragajic11#after-effects image
> > but on the content in the page still see the issue. image image - transperancy on the scroll share icons missing in the footer image
> > issues are not seen in main branch : https://main--cc--adobecom.hlx.page/creativecloud/animation/discover
> > Thankyou @Ruchika4 , verified the flow in mobile mode as well as advised and do not see the scroll functionality working https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=mwpw-136767-disable-scroll--milo--saragajic11#animate image image
> 
> Thanks for testing @spadmasa @saragajic11 looks like there is some issues with sending viewport dimensions to iframe (commerce modal) which is causing it to break in all viewports. I tested it in a test branch by removing body css file and just defining the css class name .disable-scroll Test page here https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=disable-scroll--milo--ruchika4#after-effects and it appears to be working fine for commerce modal. @spadmasa could you also please verify this. I see this fixes some other content issue you pointed out earlier too.




Verified the flow in the url provided in all view ports looks fine.
https://main--cc--adobecom.hlx.page/creativecloud/animation/discover?milolibs=disable-scroll--milo--ruchika4#after-effects
image
image

image
image

content issue is not seen
image

image
image

File: libs/blocks/modal/modal.css (hunk 1 above)
Comment (saragajic11) [reply libs/blocks/modal/modal.css]: Hi @Ruchika4 I have update the modal.css file (https://github.com/adobecom/milo/pull/1457/commits/bd85eb224bdd7c54380709f1815fecc1066ed36b) per your suggestion but I can still see that commerce modal is not opening the way it is on main branch...

Comment (Ruchika4) [reply libs/blocks/modal/modal.css]: oh ok, could you please investigate then what is causing the issue. As with current code it is working as expected and it's breaking with this PR. Thanks

**Comment by saragajic11** [comment]
Closing this one as new PR has been opened which doesn't affect content:
https://github.com/adobecom/milo/pull/1466

**Pull request closed without merging**